src/material-examples/expansion-steps/expansion-steps-example.html

```diff
@@ -0,0 +1,70 @@
+<md-accordion class="example-headers-align">
+  <md-expansion-panel [expanded]="step === 0" (opened)="openEvent(0)" hideToggle="true">
```

Comment:
Personally I think the toggles should be shown (instead of the icons) since this is the main example and you're trying to demo the features of the panel. If you insist on using icons,

1. The `color: blue` is pretty harsh and could probably be dropped
2. The icon-to-right thing cuts off the description text. Instead, unwrap the icons from their parent div, and make `mat-expansion-panel-header-description` justify-content with space-between and align-items center